apps/apps-cli/.env.example

```diff
@@ -0,0 +1,3 @@
+INSTANCE_URL=https://demo.saleor.io/graphql/
+USER_EMAIL=admin@example.com
+USER_PASSWORD=password123
```

Comment:
I'm not sure if it works like intended. This assumes we have only one env we are working with.

The important case is to target many envs, so I can install app on multiple Saleor versions